src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -106,6 +106,21 @@ type Todo @model @auth(rules: [{ allow: owner, ownerField: "author" }]) {
 }
 ```
 
+<Callout warning>
+
+  **By default, owners have full permissions on their records!**
+
+  Unless [field-level authorization](#field-level-authorization-rules) is enabled, although an owner field is automatically generated, the 'owner' of a record has full update/delete permissions, which can include updating the owner to a nonexistent or different user. If this is not desirable behavior, field-level authorization may be configured on the owner field itself.
```

Comment:
```suggestion
To prevent an owner to reassign their record to another user, protect the owner field (by default `owner: String`) with a field-level authorization rule. 
```

Reply:
Updated, reworded slightly, but looks good to me.